### PR TITLE
[WFAPI-1010] List ALL subscriptions rather than the first 100

### PR DESCRIPTION
**Pull request by emirkin**

### CODE REVIEW

File: lib/pub_sub/tasks/debug.rake [hunk 1]
```diff
@@ -29,11 +29,18 @@ namespace :pub_sub do
     desc 'List information about the queue subscriptions.'
     task subscriptions: :environment do
       puts 'Subscriptions: ', '----------'
+      puts "SERVICE\tSUBSCRIPTION\tPROTOCOL\tREGION"
       PubSub.config.regions.each do |region|
-        subs = Aws::SNS::Client.new(region: region).list_subscriptions.subscriptions
+        subs = []
+        next_token = ""
+        while true
```
Comment (gwshaw) [inline lib/pub_sub/tasks/debug.rake]: Rather than `while true` you can do `loop do` for an infinite loop.

Comment (ltran) [inline lib/pub_sub/tasks/debug.rake]: I favor `loop do` as well. also seems to be the standard based of the ruby community for infinite loops.

**Comment by gwshaw** [comment]
Only one comment, LGTM

**Comment by gwshaw** [comment]
LGTM

File: lib/pub_sub/tasks/debug.rake [hunk 2]
```diff
@@ -42,14 +43,30 @@ namespace :pub_sub do
...
   def split_name(string, delimiter = ':')
     string.to_s.split(delimiter).last
   end
+
+  def collect_all(client, collection_type)
+    elements = []
+    next_token = ""
+    loop do
+      response = client.send("list_#{collection_type}".to_sym, next_token: next_token)
+      elements.concat response.send(collection_type.to_sym)
```
Comment (ltran) [inline lib/pub_sub/tasks/debug.rake]: Do we need to whitelist this  send? or is the input a considered safe?

**Comment by ltran** [comment]
LGTM

**Pull request merged**